### PR TITLE
add j2cli to sonic_debian_extension.j2

### PR DESCRIPTION
**Pull request by VenkatCisco**
#### Why I did it
j2cli provides access to jinja library. Our platform.py requires j2cli to handle jinja template configuration files.

#### How I did it
add j2cli to sonic_debian_extension.j2 file 

#### How to verify it
install j2cli

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [x] 202006
- [x] 202012

### CODE REVIEW

**Review by lguohan — APPROVED** [review]

File: files/build_templates/sonic_debian_extension.j2 [hunk 1]
```diff
@@ -114,6 +114,9 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     python-dev     \
     python3-dev
 
+# Install j2cli for handling jinja template
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install j2cli
```
Comment (lguohan) [inline files/build_templates/sonic_debian_extension.j2:118]: it is better to specify a version == so that it won't break if j2cli upgrades.

Comment (VenkatCisco) [reply files/build_templates/sonic_debian_extension.j2]: updated j2cli with a specific version

**Comment by anshuv-mfst** [comment]
@lguohan - could you please help with approval & merge. Build failures are fixed.

**Comment by anshuv-mfst** [comment]
@lguohan - can you please approve & merge this PR.

**Review by lguohan — APPROVED** [review]

**Pull request merged**